src/app/components/BibPage/ElectronicResources.jsx

```diff
@@ -0,0 +1,17 @@
+import { Card, CardContent, CardHeading } from '@nypl/design-system-react-components'
+import generateERLinks from '../../utils/electronicResources'
+import React from 'react';
+
+const ElectronicResources = (props) => {
```

Comment:
Nitpick: destructure the prop values here instead of calling `props.electronicResources`.
```
const ElectronicResources = ({ electronicResources }) => {
...
```